guide/case-studies/walletscrutiny.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: guide
+title: WalletScrutiny
+description: This was a collaboration between WalletScrutiny and the Bitcoin Design Community to help with the redesign of the website.
+nav_order: 4
+parent: Case studies
+permalink: /guide/case-studies/wallet-scrutiny/
```

Comment:
```suggestion
permalink: /guide/case-studies/walletscrutiny/
```
The next/previous buttons on the payjoin/resources pages that link to this page are not working because they link to `.../walletscrutiny/`. If that is the final URL you want for this page, it should be updated here as well.